kde-plasma/plasma-meta/plasma-meta-5.6.49.9999.ebuild

```diff
@@ -53,7 +53,7 @@ RDEPEND="
 		$(add_plasma_dep breeze-gtk)
 		$(add_plasma_dep kde-gtk-config)
 	)
-	handbook? ( || ( $(add_kdeapps_dep khelpcenter) $(add_plasma_dep khelpcenter) ) )
+	handbook? ( $(add_kdeapps_dep khelpcenter '' ${PV}) )
```

Comment:
well in this case it is PV so that kde-apps/khelpcenter dependency can correspond to 5.6.x instead of KDE_APPS_MINIMAL which would otherwise be 14.12.